components/affirmation/Affirmation.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+
+function Affirmation(props) {
+ return (
+
+ <div className="flip_box">
+  <div className="flip_box_inner">

```

Comment:
very clear in div structure, maybe we could keep with style-components which keeps a set of styling consistent for larger web applications ✨ 

Great work on this 🌟 🎨 🖌️ 👩‍🎨